### PR TITLE
All the missing pieces in RFC 9000 that I found

### PR DESCRIPTION
**Pull request by martinthomson**

### CODE REVIEW

File: rfc9000.md [hunk 1]
```diff
@@ -1382,7 +1382,7 @@ when a client's address changes.
 
 A server in a deployment that does not implement a solution to maintain
 connection continuity when the client address changes SHOULD indicate that
-migration is not supported by using the disable_active_migration transport
```
Comment (MikeBishop) [inline rfc9000.md:1385]: @janaiyengar explicitly added this "by" in 4601579.

File: rfc9000.md [hunk 2]
```diff
@@ -1829,7 +1829,7 @@ constraints conflict.
 
 Using 0-RTT depends on both client and server using protocol parameters that
 were negotiated from a previous connection.  To enable 0-RTT, endpoints store
-the values of the server transport parameters with any session tickets it
+the value of the server transport parameters with any session tickets it
```
Comment (MikeBishop) [inline rfc9000.md:1832]: This seems inconsistent with the remainder of the paragraph and following, where we talk about "values" for the individual parameters.

**Review by MikeBishop — APPROVED** [review]

File: rfc9000.md (hunk 1 above)
Comment (janaiyengar) [reply rfc9000.md]: Yup -- @MikeBishop added "that" on line 1384, which I think makes this "by" necessary.

**Review by janaiyengar — APPROVED** [review]
one comment but rest lgtm -- thanks!

**Pull request merged**